types/floreal/index.d.ts

```diff
@@ -2,6 +2,5 @@
 // Project: https://github.com/seeschloss/floreal
 // Definitions by: mike castleman <https://github.com/mlc>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
-import FlorealDate = require('./date');
+import FlorealDate = require("./date");
```

Comment:
any particular reason all the single quotes got changed to double quotes?

Reply:
No particular, only DT default configuration uses double quotes, let me know if you want to stick to single quotes